### PR TITLE
feat/new card providers

### PR DESCRIPTION
**Pull request by pedroapfilho**
- feat(cards): adding new way to handle add_card
- feat(cards): adding boilerplate for adding card on checkout

### CODE REVIEW

**Review by plondon — APPROVED** [review]
LGTM

File: packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts [hunk 1]
```diff
@@ -1191,6 +1192,66 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
...
+        yield put(
+          A.setStep({
+            step: 'ADD_CARD_EVERYPAY'
+          })
+        )
+
+        return
+      }
+
+      // const cardAcquirers: CardAcquirer[] = yield call(api.getCardAcquirers)
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/data/components/buySell/sagas.ts:1211]: maybe to remove if not used?

File: packages/blockchain-wallet-v4-frontend/src/modals/BuySell/index.tsx [hunk 2]
```diff
@@ -176,7 +177,23 @@ class BuySell extends PureComponent<Props, State> {
...
+            {this.props.step === 'ADD_CARD_EVERYPAY' && (
+              <FlyoutChild>
+                <AddCardEverypay {...this.props} handleClose={this.handleClose} />
+              </FlyoutChild>
+            )}
+            {this.props.step === 'ADD_CARD_CHECKOUT' && (
+              <FlyoutChild>
+                {
+                  // eslint-disable-next-line no-console
+                  console.log('adasdasdasdas')
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/BuySell/index.tsx:194]: ah testing purpose ;)

**Review by milan-bc — APPROVED** [review]
🚀

**Pull request merged**